Automatic Validation ended with:
> 2024-04-26T19:56:19.0351217Z ##[error]Manifest Validation Failed
 2024-04-26T19:56:24.1005379Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.installer.yaml
@@ -1,0 +1,23 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2023.2.1.25
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSuccessCodes:
+- 1223
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: "2023.2"
+ElevationRequirement: elevatesSelf
+ReleaseDate: 2024-04-09
+Installers:
+- Architecture: x64
+  InstallerUrl: https://redirector.gvt1.com/edgedl/android/studio/install/2023.2.1.25/android-studio-2023.2.1.25-windows.exe
+  InstallerSha256: EA891F977305DFAF58DFAFED7908A6B14D7FDD9D2B024A80A13BD80B3A82C346
+ManifestType: installer
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.locale.en-US.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.locale.en-US.yaml
@@ -1,0 +1,44 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2023.2.1.25
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/studio/intro
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android Studio
+PackageUrl: https://developer.android.com/studio
+License: Android Software Development Kit License Agreement
+LicenseUrl: https://developer.android.com/studio/terms
+Copyright: Copyright © 2000–2024 Google
+CopyrightUrl: https://developer.android.com/studio/terms
+ShortDescription: The official Integrated Development Environment (IDE) for Android app development.
+Description: |-
+  Android Studio is the official Integrated Development Environment (IDE) for Android app development.
+  Based on the powerful code editor and developer tools from IntelliJ IDEA,
+  Android Studio offers even more features that enhance your productivity when building Android apps.
+Moniker: androidstudio
+Tags:
+- aab
+- adb
+- android
+- android-studio
+- apk
+- app
+- debug
+- development
+- emulator
+- fastboot
+- ide
+- intellij
+- studio
+ReleaseNotesUrl: https://developer.android.com/studio/releases
+Documentations:
+- DocumentLabel: Troubleshoot
+  DocumentUrl: https://developer.android.com/studio/troubleshoot
+- DocumentLabel: Known Issues
+  DocumentUrl: https://developer.android.com/studio/known-issues
+ManifestType: defaultLocale
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.2.1.25/Google.AndroidStudio.yaml
@@ -1,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2023.2.1.25
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.6.0

